endpoints/openrtb2/amp_auction.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 type AmpResponse struct {
-	Targeting map[string]string `json:"targeting"`
+	Targeting map[string]string             `json:"targeting"`
+	Debug     *openrtb_ext.ExtResponseDebug `json:"debug,omitempty"`
```

Comment:
This could just be of type `openrtb.RawJSON` and avoid the extraneous unmarshall->remarshall, though that only happens on a debug call and this ensures the interface.